File: Java/exercises/RandStats/rand-stats.pl
#: rand-stats.pl

use v6-alpha;

my ($max, $min, $nbig);

for 1..100 {
    my $e = rand 100;
    $max = $e if !$max.defined or $max < $e;
    $min = $e if !$min.defined or $min > $e;
    $nbig++ if $e > 60;
}
say "Maximum is $max.";
say "Minimum is $min.";
say "Count of nums greater than 60 is $nbig.";

File: Java/exercises/RandStats/RandStats.java
//: RandStats.java

public class RandStats {
    public static void main(String args[]) {
        double max = 0, min = 0, nbig = 0;
        for (int i = 0; i < 100; i++) {
            double e = Math.random() * 100;
            if (i == 0 || max < e) max = e;
            if (i == 0 || min > e) min = e;
            if (e > 60) nbig++;
        }
        say("Maximum is " + min);
        say("Minimum is " + max);
        say("Count of nums greater than 60 is " + nbig);
    }
    private static void say(String s) {
        System.out.println(s);
    }
}
